src/main/java/com/project/bookstore/BookStoreApplication.java

```diff
@@ -1,12 +1,22 @@
 package com.project.bookstore;
 
+import com.project.bookstore.service.RoleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class BookStoreApplication {
+@RequiredArgsConstructor
+    public class BookStoreApplication implements CommandLineRunner {
+    private final RoleService roleService;
 
     public static void main(String[] args) {
         SpringApplication.run(BookStoreApplication.class, args);
     }
+
+    @Override
+    public void run(String... args) throws Exception {
+        roleService.fillRolesTable();
+    }
```

Comment:
Why do you do it here? You can fill it by liquibase

Reply:
Want to have more flexibility in application.

Reply:
This is hardcode, not a flexibility )
Delete this method and all related code, please